### PR TITLE
Edits to text in Phrase Suggester doc

### PR DESCRIPTION
**Pull request by dmeiss**
Minor text edits for phrasing, punctuation, and capitalization.

### CODE REVIEW

File: docs/reference/search/suggesters/phrase-suggest.asciidoc [hunk 1]
```diff
@@ -141,21 +141,21 @@ The response contains suggestions scored by the most likely spell correction fir
...
-    based on the specified `field`. If the field uses a `shingle` filter the
+    based on the specified `field`. If the field uses a `shingle` filter, the
     `gram_size` is set to the `max_shingle_size` if not explicitly set.
 
 `real_word_error_likelihood`::
-    the likelihood of a term being a
+    The likelihood of a term being a
     misspelled even if the term exists in the dictionary. The default is
-    `0.95` corresponding to 5% of the real words are misspelled.
+    `0.95`, meaning 5% of the real words are misspelled.
```
Comment (dmeiss) [inline docs/reference/search/suggesters/phrase-suggest.asciidoc]: I was a bit unsure about the accuracy of this edit.

File: docs/reference/search/suggesters/phrase-suggest.asciidoc [hunk 2]
```diff
@@ -167,33 +167,33 @@ The response contains suggestions scored by the most likely spell correction fir
...
 `max_errors`::
-    the maximum percentage of the terms that at most
+    The maximum percentage of the terms 
     considered to be misspellings in order to form a correction. This method
     accepts a float value in the range `[0..1)` as a fraction of the actual
     query terms or a number `>=1` as an absolute number of query terms. The
-    default is set to `1.0` which corresponds to that only corrections with
-    at most 1 misspelled term are returned.  Note that setting this too high
-    can negatively impact performance. Low values like `1` or `2` are recommended
+    default is set to `1.0`, meaning only corrections with
```
Comment (dmeiss) [inline docs/reference/search/suggesters/phrase-suggest.asciidoc]: I was a bit unsure about the accuracy of this edit.

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

File: docs/reference/search/suggesters/phrase-suggest.asciidoc (hunk 1 above)
Comment (jimczi) [reply docs/reference/search/suggesters/phrase-suggest.asciidoc]: ++, the change looks good

File: docs/reference/search/suggesters/phrase-suggest.asciidoc (hunk 2 above)
Comment (jimczi) [reply docs/reference/search/suggesters/phrase-suggest.asciidoc]: It looks good to me

**Review by jimczi — APPROVED** [review]
LGTM, thanks @dmeiss !

**Pull request merged**